docs/account_tutorial.md

```diff
@@ -135,10 +42,8 @@ These values 32bits flags. So 8 x 32bits values makes 256 bits available to stor
 
 Each bit means:
 
-``` cpp
-  0  -  Not yet shown
-  1  -  Shown
-```
+0  -  Not yet shown
+1  -  Shown
 
```

Comment:
these lines appear one next-to the other, try to put a newline in between, that means in markdown 2 spaces at the end of the first line

Reply:
Thanks